src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'Advanced Data Scrubbing (Preview)'
+sidebar_order: 1
+keywords: ["pii", "gdpr", "personally identifiable data", "compliance"]
+---
+
+In addition to using [`beforeSend`]({% link _documentation/data-management/sensitive-data.md %}#custom-event-processing-in-the-sdk) in your SDK or our [server-side data scrubbing features]({% link _documentation/data-management/sensitive-data.md %}#server-side-scrubbing) to redact sensitive data, we are currently beta-testing ways to give you more granular control over what exactly our server-side data scrubbing does to your events. Additional functionality includes:
+
+* Ability to define custom regular expressions to match on sensitive data
+* More fine-grained tuning on which parts of an event to scrub
+* Partial removal or hashing of sensitive data instead of just deletion
+
+## Overview
+
+**Advanced Data Scrubbing is currently only available if your organization is an Early Adopter.** To become one, go to your organization settings and enable the "Early Adopter" option to get access to new features before they're fully released. This option can be just as easily disabled as it can be enabled.
+
+You will then get access to one new option in both organization settings and the settings of each project called "Advanced datascrubber configuration". Its content is a piece of JSON-based configuration in the format described in the next section.
```

Comment:
Suggest "Early adopters have access to a new option ("Advanced datascrubber configuration") in both organization settings as well as the setting of each project, which creates a JSON-based configuration as described in the following section."